metaflow/plugins/kfp/kfp.py

```diff
@@ -959,9 +960,13 @@ def build_kfp_dag(
                     envs = kfp_component.environment_decorator.attributes[
                         "kubernetes_vars"
                     ]
-                    for env in envs:
+                    for env in envs if envs else []:
                         container_op.container.add_env_variable(env)
 
+                    env_vars = kfp_component.environment_decorator.attributes["vars"]
+                    for name, value in env_vars.items() if env_vars else []:
+                        container_op.container.add_env_variable(V1EnvVar(name, value))
+
```

Comment:
Does this change accomplish the same as https://github.com/zillow/metaflow/blob/feature/kfp/metaflow/plugins/kfp/kfp_decorator.py#L91-L95 ? Let's just keep one of them.

Seeing your implementation here do we want to deprecate `kubernetes_vars` and use only `vars`? `container_op.container.add_env_variable(V1EnvVar(name, value))` seems to work in both cases right?